package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "firebase": "^10.7.2",
```

Comment:
Mě to hází 2 errory, prosím o fix
<img width="1194" alt="Screenshot 2024-01-28 at 22 31 46" src="https://github.com/DenisaZem/my-lexicon/assets/61468175/ab4552ed-abd2-4a22-9d36-730b607db5cd">